### ISSUE TITLE
Update to 1.21 please

### ISSUE THREAD

**Issue by EGOIST1372** [Lizard-Of-Oz/Searchlight#13]

**Comment by antiwesley** [comment]
I agree. This is a very awesome mod, and I'd love to see it updated AND ported to NeoForge. :)

**Comment by MankyHamster** [comment]
I'd love to see this updated to 1.21.1 too ... I use it quite a bit over my modern and futuristic builds so our server is 'stuck' in the past at the moment
